### PR TITLE
Corrected task description for CSS tables

### PR DESCRIPTION
**Pull request by yahyavaleo**
### Description

On page [Test your skills: Tables](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Tables_tasks), the description for the task and the finished table do not match. The finished table requires more steps than are suggested in the task description:

- It requires the table footer heading to be aligned right.
- It requires the table footer data to be aligned left.

Furthermore, the task no. 4 is ambiguous. It does not specify where to apply the top and bottom borders (to the table or the footer)?

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/tables_tasks/index.md [hunk 1]
```diff
@@ -21,7 +21,10 @@ Our finished table will look like the image below. There are a number of ways th
 - Add padding of `0.3em` to the table headings and data and align them at the top of their cells.
 - Align headings and data for columns containing numbers right.
 - Align headings and data for columns containing text left.
-- Add a 1px top and bottom solid border with the hex color `#999`, plus a 1px solid border of the same color above the footer.
+- Align heading of the table footer right.
```
Comment (chrisdavidmills) [inline files/en-us/learn/css/building_blocks/tables_tasks/index.md:24]:
```suggestion
- Align the heading of the table footer right.
```

File: files/en-us/learn/css/building_blocks/tables_tasks/index.md [hunk 2]
```diff
@@ -21,7 +21,10 @@ Our finished table will look like the image below. There are a number of ways th
...
+- Align data of the table footer left.
```
Comment (chrisdavidmills) [inline files/en-us/learn/css/building_blocks/tables_tasks/index.md:25]:
```suggestion
- Align the data of the table footer left.
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@yahyavaleo thank you so much for these fixes. I've made a couple of small suggestions for grammar fixes.

**Comment by yahyavaleo** [comment]
@chrisdavidmills Thank you for your suggestions (I committed them). Please review again.

**Review by chrisdavidmills — APPROVED** [review]
@yahyavaleo ooops, my apologies; I missed your update on this one. Approving; thanks again for the great work!

Thanks for the nudge @Josh-Cena.

**Pull request merged**